.travis.yml

```diff
@@ -3,19 +3,30 @@ dist: trusty
 
 language: generic
 
+env:
```

Comment:
Travis docs claim this line is required for `allow_failures` to work, but it also adds an extra job with no env vars that fails, hmm